plugins/include/open62541/plugin/log_stdout.h

```diff
@@ -8,9 +8,35 @@
 #define UA_LOG_STDOUT_H_
 
 #include <open62541/plugin/log.h>
+#include <open62541/server_config.h>
+#include <open62541/client_config.h>
 
 _UA_BEGIN_DECLS
 
+/* ANSI escape sequences for color output taken from here:
+ * https://stackoverflow.com/questions/3219393/stdlib-and-colored-output-in-c*/
+
+#ifdef UA_ENABLE_LOG_COLORS
```

Comment:
Keep the defines in the .c file.
They are internal and not part of the API.

Reply:
Moved it latest commit